source/adam_parser.cpp

```diff
@@ -86,15 +83,15 @@ namespace adobe {
 
 adam_parser::adam_parser(std::istream& in, const line_position_t& position)
     : expression_parser(in, position) {
-    set_keyword_extension_lookup(boost::bind(&keyword_lookup, _1));
+    set_keyword_extension_lookup(std::bind(&keyword_lookup, _1));
```

Comment:
Are these still necessary? Will this line compile without the bind?

(And so throughout this file.)